controllers/FollowController.js

```diff
@@ -0,0 +1,143 @@
+const {
+    getUserById,
+    followUser,
+    unfollowUser,
+    getUserFollowee,
+    followers
+  } = require('../lib/modelManagers/followModel'),
+  {
+    defaultResponse,
+    failureResponse,
+    successResponse
+  } = require('../lib/utils/helper_function');
+
+const {
+  OK_CODE,
+  RESOURCE_CREATED_CODE,
+  BAD_REQUEST_CODE,
+  ALREADY_FOLLOWING_AUTHOR,
+  FOLLOW_SELF,
+  NOT_FOUND_CODE,
+  NOW_FOLLOWING_USER,
+  UNFOLLOW_USER_MESSAGE,
+  MY_FOLLOWEES,
+  MY_FOLLOWERS,
+  NOT_FOUND_CODE_MESSAGE,
+  NO_FOLLOWER,
+  NO_FOLLOWEE
+} = require('../constants/index');
+
+/**
+ * @description Class contains follow and unfollow user
+ * @class FollowUsersController
+ */
+class FollowUsersController {
+  /**
+  *
+  *
+  * @static
+  * @param {any} req Request Object
+  * @param {any} res Response Object
+  * @returns {*} - Return a new follower
+  * @memberOf FollowUsersController
+  */
+  static async follow(req, res) {
+    try {
+      const followeeId = req.decodedToken.payLoad;
+      const { id } = req.params;
+      const user = await getUserById(id);
+      if (user) {
```

Comment:
I believe we already have a user verification middleware, why are you still verifying a user's existence here?

Reply:
Fixed